sdk/keyvault/azsecrets/utils_test.go

```diff
@@ -87,9 +87,9 @@ func createClient(t *testing.T) (*Client, error) {
 
 	var cred azcore.TokenCredential
 	if recording.GetRecordMode() == "record" {
-		tenantId := lookupEnvVar("KEYVAULT_TENANT_ID")
-		clientId := lookupEnvVar("KEYVAULT_CLIENT_ID")
-		clientSecret := lookupEnvVar("KEYVAULT_CLIENT_SECRET")
+		tenantId := lookupEnvVar("KEYVAULT/AZSECRETS_TENANT_ID")
```

Comment:
This is a weird format and doesn't work in PS.

Reply:
This passed the live pipeline, which I believe uses PS to deploy the resources. However, there is a bug in the deploy script for go that ben is aware of and this will go away in the future.

Reply:
It passes in the live test pipeline because presumably the code works when `tenantId` equals an empty string. I'm going to update the deployment script to parse service directories with multiple segments.

Reply:
Given that this variable reference does not work, you should change this string to the one that will work `AZSECRETS_TENANT_ID` in case you merge before my fix.